bindings/action_system.json

```diff
@@ -69,7 +76,10 @@
           "spend": "Spend"
         }
       },
-      "spend": {}
+      "spend": {},
```

Comment:
This should return the `Outputs` rather than another method, since it's returned at this point on the Rust side